tests/Jellyfin.Api.Tests/Jellyfin.Api.Tests.csproj

```
@@ -15,6 +19,13 @@
 
   <ItemGroup>
     <ProjectReference Include="../../MediaBrowser.Api/MediaBrowser.Api.csproj" />
+    <ProjectReference Include="..\..\Jellyfin.Api\Jellyfin.Api.csproj" />
```

```suggestion
    <ProjectReference Include="../../Jellyfin.Api/Jellyfin.Api.csproj" />
```
Technically it doesn't matter, but consistency ;)